src/sdk/namespace_fs.js

```diff
@@ -869,7 +872,7 @@ class NamespaceFS {
                 if (r.common_prefix) {
                     res.common_prefixes.push(r.key);
                 } else {
-                    obj_info = this._get_object_info(bucket, r.key, r.stat, 'null', false, true);
+                    obj_info = this._get_object_info(bucket, r.key, r.stat, 'null', false, r.is_latest);
```

Comment:
Unrelated to your changes but you change this line anyway, 
I remember I wrote myself to change this 'null' to a boolean, does it make sense?

Reply:
its the return_version_id value its ignored, so I removed it